### ISSUE TITLE
data-header change

### ISSUE THREAD

**Issue by qxygene** [snapappointments/bootstrap-select#1187]
i cant change header dynamicly;

```
        var test = $("#a1").attr('data-header');
        console.log(test);
        var test2 = $("#a1").attr('data-header', 'new header'); 
        $("#a1").selectpicker('render');
                    $("#a1").selectpicker('refresh');
        console.log(test2);
```

i can see new header on console but not on select.

**Comment by qxygene** [comment]
something to do wthi refresh function;
header = this.options.header ? this.$menu.find('.popover-title')[0].cloneNode(true) : null,
?

**Comment by wesleyb-io** [comment]
I believe the problem you experience can be solved in the same way as this issue was solved: [data-icon doesn't update after refresh / render](https://github.com/silviomoreto/bootstrap-select/issues/1079#issuecomment-159264365).

**Comment by caseyjhol** [comment]
Fixed in v1.13.6.
